### DISCUSSION TITLE
NC 24

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#559] [Questions] [answered]
Hello, my question is when can we expect the NC 24 update?

**Comment by szaimen** [comment, marked as answer]
Hello, we will wait until all important apps are compatible with Nextcloud 24 and will wait at least until 24.0.1 is out.
I'll add this to the FAQ section.

**Reply by szaimen** [reply]
documented in https://github.com/nextcloud/all-in-one/pull/562

**Comment by szaimen** [comment]
This is now released with 1.7.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
